components/collection/CollectionInfo.vue

```diff
@@ -28,7 +28,11 @@
             <template #content>
               <div class="bg-background-color border py-2 px-4 text-xs">
                 Recipient Address:
+                <div v-if="recipient.startsWith('0x')" class="break-all">
```

Comment:
I believe we already had some utils to check if the address is on evm. CMIIW
cc @hassnian

Reply:
`import { isEthereumAddress } from '@polkadot/util-crypto'`